packages/gp2-components/src/templates/OutputForm.tsx

```diff
@@ -134,6 +154,25 @@ const OutputForm: React.FC<OutputFormType> = ({
       value: author.id,
     })) || [],
   );
+  const [newTags, setNewTags] = useState<gp2Model.KeywordDataObject[]>(
+    tags || [],
+  );
+
+  const identifierType: gp2Model.OutputIdentifierType = doi
+    ? gp2Model.OutputIdentifierType.DOI
+    : rrid
+    ? gp2Model.OutputIdentifierType.RRID
+    : accessionNumber
+    ? gp2Model.OutputIdentifierType.AccessionNumber
+    : title // if it's editing
```

Comment:
I'm curious about this `title` here. If it's editing and the values of doi, rrid and accessionNumber are undefined, shouldn't be none directly instead of this title?
<img width="527" alt="Screenshot 2023-09-27 at 13 07 02" src="https://github.com/yldio/asap-hub/assets/16595804/3eb0bd04-53e5-4042-929e-cd6ce329ffbe">

Reply:
yes, but if it's creating a new output, we want to have nothing selected (empty). in that case, title is not defined.

Reply:
I see, it makes sense